build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
   id 'groovy'
   id 'maven'
   id 'java-gradle-plugin'
-  id 'com.github.sherter.google-java-format' version '0.6'
+  id 'com.github.sherter.google-java-format' version '0.8'
```

Comment:
GoogleJavaFormat plugin 0.9 (latest) complains minimum Java version:

> com/google/googlejavaformat/java/JavaFormatterOptions$Style has
been compiled by a more recent version of the Java Runtime (class
file version 55.0), this version of the Java Runtime only recognizes
class file versions up to 52.0

Therefore choosing the second latest version 0.8 that does not
complain about the Java version.